warp/src/main/java/com/schibsted/nmp/warp/components/WarpPill.kt

```diff
@@ -73,7 +75,12 @@ fun WarpPill(
         colors = pillColors,
         border = null,
         modifier = modifier,
-        trailingIcon = if (closable) {
+        trailingIcon = if (icon != null) {
+            {
+                icon()
```

Comment:
this is again the same discussion we've had a few times, but I think it might be better to only allow for providing the icon ressource and contentDescription?
I understand this would increase even more the number of params, but I feel warp should ensure color and size consistency on this...?

Reply:
Yeap, good point. Ill check with design and update :+1: